indico/modules/events/abstracts/lists.py

```diff
@@ -88,6 +88,9 @@ def _get_filters_from_request(self):
                 options = [x if x != 'None' else None for x in request.form.getlist(f'field_{field.id}')]
                 if options:
                     filters['fields'][str(field.id)] = options
+        # Ensure enum filters remain as integers
+        for idx, value in enumerate(filters['items'].get('state', [])):
+            filters['items']['state'][idx] = int(value)
```

Comment:
The fact that we always add a state filter here - even if it didn't exist before - isn't a problem?

Reply:
What I meant is adding `filters['items']['state']` even if it wasn't there before. Empty list could mean "nothing", but I didn't look more closely at the code. Just wanted to bring up something to check. So if it all works fine no need to change anything